src/openforms/conf/base.py

```diff
@@ -1027,13 +1027,11 @@
     "'self'",
 ] + config("CSP_EXTRA_DEFAULT_SRC", default=[], split=True)
 
-# CORS_ALLOWED_ORIGINS is included because we (likely) need to redirect back to those
-# third party domains that are embedding the SDK after login. Chrome in particular
-# validates the entire redirect chain, see: https://stackoverflow.com/a/69439102
-CSP_FORM_ACTION = (
-    ["'self'"]
-    + CORS_ALLOWED_ORIGINS
-    + config("CSP_EXTRA_FORM_ACTION", default=[], split=True)
+# Allow any 'https:' host, as we don't know in advance which target is used by eHerkenning.
+# Behavior is also different between browsers regarding redirects:
+# https://github.com/w3c/webappsec-csp/issues/8
```

Comment:
Can you please restore the stackoverflow link? I think it illustrates the problem much better with practical examples compared to the github link. It's fine to add the github link for an "authorative" source, but then it should be an additional reference.